pkg/broker/broker.go

```diff
@@ -111,32 +108,35 @@ type DevBroker interface {
 
 // AnsibleBroker - Broker using ansible and images to interact with oc/kubernetes/etcd
 type AnsibleBroker struct {
-	dao           *dao.Dao
-	log           *logging.Logger
-	clusterConfig apb.ClusterConfig
-	registry      []registries.Registry
-	engine        *WorkEngine
-	brokerConfig  Config
+	dao          *dao.Dao
+	log          *logging.Logger
+	registry     []registries.Registry
+	engine       *WorkEngine
+	brokerConfig Config
 }
 
 // NewAnsibleBroker - Creates a new ansible broker
-func NewAnsibleBroker(dao *dao.Dao, log *logging.Logger, clusterConfig apb.ClusterConfig,
-	registry []registries.Registry, engine WorkEngine, brokerConfig Config,
+func NewAnsibleBroker(dao *dao.Dao, log *logging.Logger,
+	registry []registries.Registry, engine WorkEngine, brokerConfig *config.Config,
 ) (*AnsibleBroker, error) {
 	broker := &AnsibleBroker{
-		dao:           dao,
-		log:           log,
-		clusterConfig: clusterConfig,
-		registry:      registry,
-		engine:        &engine,
-		brokerConfig:  brokerConfig,
+		dao:      dao,
+		log:      log,
+		registry: registry,
+		engine:   &engine,
+		brokerConfig: Config{
+			DevBroker:          brokerConfig.GetBool("dev_broker"),
+			LaunchApbOnBind:    brokerConfig.GetBool("launch_apb_on_bind"),
+			BootstrapOnStartup: brokerConfig.GetBool("bootstrap_on_startup"),
+			Recovery:           brokerConfig.GetBool("recovery"),
+			OutputRequest:      brokerConfig.GetBool("output_request"),
+			SSLCertKey:         brokerConfig.GetString("ssl_cert_key"),
+			SSLCert:            brokerConfig.GetString("ssl_cert"),
+			RefreshInterval:    brokerConfig.GetString("refresh_interval"),
+			AutoEscalate:       brokerConfig.GetBool("auto_escalate"),
+			ClusterURL:         brokerConfig.GetString("cluster_url"),
+		},
 	}
-
-	err := broker.Login()
```

Comment:
We are able to get rid of this because the initialization of the clients does the things that we need. I think this code has not been needed for a long time.

Reply:
Might be better to add it back and remove in an explicit PR?

Reply:
@eriknelson I just realized while adding it back why I removed it. I removed the clusterConfig from the broker object causing this to be a problem where it is with this PR.

I did not add it back because this will just initialize the OC client, while everything should no longer be using the local oc client.

Reply:
I've been commenting this out locally for a few days now because it messes up the local k8s deployment.  I don't think we need it anymore. Removing it will also be one less thing we need to add to the runtime pkg.

Reply:
Should we also be removing the Config Values as well (`Host` and `BearerTokenFile`)

Reply:
Do we know of folks wanting to use their own bearer tokens instead of using auth that k8s provides?  To me, I think it would be safe to remove it. We could always add it back in later if that is a valid use case.